oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/web/OAuth2TokenRevocationEndpointFilter.java

```diff
@@ -61,12 +59,15 @@
 	private static final String DEFAULT_TOKEN_REVOCATION_ENDPOINT_URI = "/oauth2/revoke";
 
 	private final AuthenticationManager authenticationManager;
+	private AuthenticationConverter tokenRevocationAuthenticationConverter;
 	private final RequestMatcher tokenRevocationEndpointMatcher;
-	private final Converter<HttpServletRequest, Authentication> tokenRevocationAuthenticationConverter =
-			new DefaultTokenRevocationAuthenticationConverter();
 	private final HttpMessageConverter<OAuth2Error> errorHttpResponseConverter =
 			new OAuth2ErrorHttpMessageConverter();
 
+	private AuthenticationSuccessHandler authenticationSuccessHandler = (request, response, authentication) -> {};
```

Comment:
Initialize the handlers with method references the same as in `OAuth2TokenEndpointFilter` and `OAuth2AuthorizationEndpointFilter`